testing-utils/src/lib/fixtures/report.mock.ts

```diff
@@ -1,4 +1,4 @@
-import { type Report } from '../../../../packages/models/src';
+import { type Report } from '@code-pushup/models';
```

Comment:
```suggestion
import type { Report } from '@code-pushup/models';
```

Reply:
Actually, I didn't mean to do that (circular dependencies later...). But fixed the `type` nesting.